engine/api/v2_rbac.go

```diff
@@ -54,3 +124,332 @@ func (api *API) postImportRbacHandler() ([]service.RbacChecker, service.Handler)
 			return service.WriteMarshal(w, req, nil, http.StatusCreated)
 		}
 }
+
+func (a *API) FillRBACWithNames(ctx context.Context, r *sdk.RBAC) error {
```

Comment:
This looks like loader func in dao, can we move this functions on a dedicated struct that can be RBACLoader, this struct can also contains the map used to cache groups, users, etc.